{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-17T21:29:00.000+0000",
      "latitude": 51.411346363517694,
      "estimatedRestoration": "2022-09-18T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3827'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3827",
      "reference": "KA3827",
      "maxLongitude": -1.0315,
      "minLatitude": 51.4072,
      "name": "RG30 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG30 3TG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0403,
              51.4159
            ],
            [
              -1.0315,
              51.4159
            ],
            [
              -1.0315,
              51.4072
            ],
            [
              -1.0403,
              51.4072
            ],
            [
              -1.0403,
              51.4159
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-18T07:30:00.000+0000",
      "UUID": "6dbcb9d7-44d2-4bb8-aac0-f89f27bef6b8",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.0403,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4159,
      "longitude": -1.0387678187870564,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T00:45:00.000+0000",
      "latitude": 51.4499537644994,
      "estimatedRestoration": "2022-09-18T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3844'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3844",
      "reference": "KA3844",
      "maxLongitude": -1.0421,
      "engineerOnSiteTime": "2022-09-18T01:50:00.000+0000",
      "minLatitude": 51.4466,
      "name": "RG31 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG30 4PT",
        "RG30 4PU",
        "RG31 4DR",
        "RG31 4DU",
        "RG31 4EE",
        "RG31 4EF",
        "RG31 4EH",
        "RG31 4EJ",
        "RG31 4EL",
        "RG31 4EN",
        "RG31 4ER",
        "RG31 4QZ",
        "RG31 4SQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0513,
              51.4538
            ],
            [
              -1.0421,
              51.4538
            ],
            [
              -1.0421,
              51.4466
            ],
            [
              -1.0513,
              51.4466
            ],
            [
              -1.0513,
              51.4538
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "1237e7e2-3bb3-43f2-b79d-a5f6af8a6d5c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.0513,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4538,
      "longitude": -1.0471388584325099,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T04:43:00.000+0000",
      "latitude": 50.564469337215556,
      "estimatedRestoration": "2022-09-18T16:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3854'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3854",
      "reference": "KA3854",
      "maxLongitude": -2.4398,
      "engineerOnSiteTime": "2022-09-18T05:45:00.000+0000",
      "minLatitude": 50.5627,
      "name": "DT5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "DT5 1EN",
        "DT5 1EP",
        "DT5 1ES",
        "DT5 1EU",
        "DT5 1EW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.4463,
              50.5671
            ],
            [
              -2.4398,
              50.5671
            ],
            [
              -2.4398,
              50.5627
            ],
            [
              -2.4463,
              50.5627
            ],
            [
              -2.4463,
              50.5671
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-18T06:00:00.000+0000",
      "UUID": "b2db4175-c346-42c0-8fdc-deff86e9259c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.4463,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.5671,
      "longitude": -2.4431154913923834,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T05:30:00.000+0000",
      "latitude": 50.78431715051607,
      "estimatedRestoration": "2022-09-18T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3865'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3865",
      "reference": "KA3865",
      "maxLongitude": -0.6689,
      "minLatitude": 50.7832,
      "name": "PO21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO21 1BN",
        "PO21 1EG",
        "PO21 1EH",
        "PO21 1EQ",
        "PO21 1EX",
        "PO21 1EY",
        "PO21 1FJ",
        "PO21 1HB",
        "PO21 1HG",
        "PO21 1RE",
        "PO21 1RF",
        "PO21 1RJ",
        "PO21 1RQ",
        "PO21 1SR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6729,
              50.7854
            ],
            [
              -0.6689,
              50.7854
            ],
            [
              -0.6689,
              50.7832
            ],
            [
              -0.6729,
              50.7832
            ],
            [
              -0.6729,
              50.7854
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-18T07:30:00.000+0000",
      "UUID": "eb61e174-a1fe-4ae7-b62e-2434e8aee751",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.6729,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7854,
      "longitude": -0.6712339691839961,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T07:00:00.000+0000",
      "latitude": 50.83595727949864,
      "estimatedRestoration": "2022-09-18T19:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '210801'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/210801",
      "reference": "210801",
      "maxLongitude": -0.7757,
      "minLatitude": 50.835,
      "name": "PO19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO19 1EB",
        "PO19 1HA",
        "PO19 1HL",
        "PO19 1JX",
        "PO19 1TJ",
        "PO19 1TL",
        "PO19 1TQ",
        "PO19 1UA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7791,
              50.8365
            ],
            [
              -0.7757,
              50.8365
            ],
            [
              -0.7757,
              50.835
            ],
            [
              -0.7791,
              50.835
            ],
            [
              -0.7791,
              50.8365
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "300ef09e-98b4-4b68-8f51-e9f64899425f",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.7791,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8365,
      "longitude": -0.7775573216571021,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T07:18:00.000+0000",
      "latitude": 50.85014490669801,
      "estimatedRestoration": "2022-09-18T08:18:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3888'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3888",
      "reference": "KA3888",
      "name": "BH21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH21 6FA",
        "BH21 6FB",
        "BH21 6PA",
        "BH21 6PB",
        "BH21 6QA",
        "BH21 6QP",
        "BH21 6QT",
        "BH21 6QU",
        "BH21 6QX",
        "BH21 6QY",
        "BH21 6QZ",
        "BH21 6RA",
        "BH21 6RB",
        "BH21 6RD",
        "BH21 6RE",
        "BH21 6RF",
        "BH21 6RG",
        "BH21 6RH",
        "BH21 6RJ",
        "BH21 6RL",
        "BH21 6RN",
        "BH21 6RP",
        "BH21 6RQ",
        "BH21 6RR",
        "BH21 6RT",
        "BH21 6RU",
        "BH21 6RW",
        "BH21 6RX",
        "BH21 6RY",
        "BH21 6RZ",
        "BH21 6SA",
        "BH21 6SF",
        "BH21 6SG",
        "BH21 6SH",
        "BH21 6SJ",
        "BH21 6SL",
        "BH21 6SN",
        "BH21 6SP",
        "BH21 6SQ",
        "BH21 6SR",
        "BH21 6SU",
        "BH21 6SW",
        "BH21 6SX",
        "BH21 6SY",
        "BH21 6SZ",
        "BH21 6UD",
        "BH21 6UJ",
        "BH21 6UZ",
        "BH21 6XX",
        "BH21 6YT",
        "BH21 6YU",
        "BH21 6YX",
        "BH21 6YY",
        "BH21 6YZ",
        "BH24",
        "BH24 2DA",
        "BH24 2EL",
        "BH24 2ET",
        "BH31"
      ],
      "location": {
        "coordinates": [
          -1.8767055142702798,
          50.85014490669801
        ],
        "type": "Point"
      },
      "UUID": "7515edf9-2b44-49d6-91d8-aa7c04399f10",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8767055142702798,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-18T07:14:00.000+0000",
      "latitude": 57.21672690164344,
      "estimatedRestoration": "2022-09-18T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KA3889'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KA3889",
      "reference": "KA3889",
      "maxLongitude": -2.8048,
      "minLatitude": 57.1864,
      "name": "AB33 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB33 8QY",
        "AB33 8RS",
        "AB33 8RT",
        "AB33 8RU",
        "AB33 8RX",
        "AB33 8SA",
        "AB33 8SB",
        "AB33 8SE",
        "AB33 8SH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.8974,
              57.2426
            ],
            [
              -2.8048,
              57.2426
            ],
            [
              -2.8048,
              57.1864
            ],
            [
              -2.8974,
              57.1864
            ],
            [
              -2.8974,
              57.2426
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-18T08:15:00.000+0000",
      "UUID": "6944c3e1-1f4f-406b-ae9a-91956149089f",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.8974,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 57.2426,
      "longitude": -2.8635794835305437,
      "resolved": false
    }
  ]
}
